### PR TITLE
Fix config.json module file patterns

### PR DESCRIPTION
**Pull request by m-dango**

### CODE REVIEW

**Review by SleeplessByte — APPROVED** [review]

File: config.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
     "test": [
       "%{kebab_slug}.rakutest"
     ],
     "example": [
-      "%{kebab_slug}.rakumod"
+      ".meta/solutions/%{pascal_slug}.rakumod"
     ],
     "exemplar": [
-      "%{kebab_slug}.rakutest"
+      ".meta/solutions/%{kebab_slug}.rakutest"
```
Comment (SleeplessByte) [inline config.json:31]:
```suggestion
      ".meta/solutions/%{kebab_slug}.rakumod"
```

I think this should be `.rakumod`. Exemplar is "the idiomatic, one true exemplary implemetentation"

Comment (m-dango) [reply config.json:31]: Ah, I think I've misunderstood what exemplar is for. I'm not sure if its applicable for this track? Is there an example of this type of file on another track?

Comment (SleeplessByte) [reply config.json:31]: It's applicable if you have concept exercises. 

`exemplar`: https://github.com/exercism/javascript/blob/main/exercises/concept/lasagna/.meta/exemplar.js
Meant to be a canonical idiomatic solution for **concept** exercises only.

`example`: https://github.com/exercism/javascript/blob/main/exercises/practice/accumulate/.meta/proof.ci.js
Meant to be a proof that the exercise is solvable.

Comment (m-dango) [reply config.json:31]: Ah I see, thanks for the info!

**Review by SleeplessByte — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**